app/src/main/res/values/strings.xml

```diff
@@ -30,8 +30,9 @@
     <string name="check_network_settings">Please check your network settings.</string>
     <string name="notification_no_internet_title">No Internet Connection</string>
     <string name="notification_no_internet_text">Please check your network settings.</string>
-
+    <string name="trip_copied_message">%1$s copied successfully!</string>
```

Comment:
I think this message is not clear enough. From a user's perspective that is new to the application, they press the button, a toast appears and that's it. Maybe it would be better to say something like `Trip copied succesfully, you can find it in the overview screen` or something similar.